.gitmodules

```diff
@@ -1,8 +1,8 @@
-[submodule "cpp/nvgraph/cpp/thirdparty/cub"]
-	path = cpp/nvgraph/cpp/thirdparty/cub
-	url = https://github.com/NVlabs/cub.git
-	branch = 1.8.0
-[submodule "cpp/nvgraph/external/cusp"]
-	path = cpp/nvgraph/external/cusp
+[submodule "thirdparty/cusp"]
+	path = thirdparty/cusp
```

Comment:
Is this really needed when using the "NVGRAPH_LIGHT"? I don't recall any feature exposed in cugraph needing cusp. I think it worth trying to removing it.

Reply:
Yes, this compiles without thirdparty/cusp, and I removed this.